src/Microsoft.Azure.SignalR.Management/WebsocketsHubLifetimeManager.cs

```diff
@@ -118,9 +118,14 @@ public Task<bool> IsUserInGroup(string userId, string groupName, CancellationTok
             return WriteAckableMessageAsync(message);
         }
 
+        public Task CloseConnectionAsync(string connectionId, string reason, CancellationToken cancellationToken)
+        {
+            throw new NotImplementedException();
```

Comment:
let's not enable this until both mode support it

Reply:
This is not enabled until `ServiceHubContext` is public